dashboard/client/src/pages/job/JobDetail.tsx

```diff
@@ -114,8 +118,22 @@ export const JobDetailChartsPage = ({
               },
             },
             {
-              label: "Logs",
-              content: <JobLogsLink job={job} newIA />,
+              label: "Actions",
+              content: (
+                <div>
+                  <JobLogsLink job={job} newIA />
+                  <br />
+                  <CpuProfilingLink
+                    pid={job.driver_info?.pid}
+                    ip={job.driver_info?.node_ip_address}
+                  />
+                  <br />
+                  <CpuStackTraceLink
+                    pid={job.driver_info?.pid}
+                    ip={job.driver_info?.node_ip_address}
+                  />
```

Comment:
I don't know if it's obvious this refers to profiling for the driver? Should we include text that says Driver cpu profile. or Driver stack trace?

Reply:
Good point. I thought it was for this actor..

Reply:
Hmm why isn't it obvious though? It is in the job row?